src/m365/spo/commands/spo-search.ts

```diff
@@ -313,9 +331,32 @@ class SpoSearchCommand extends SpoCommand {
   }
 
   private getSelectPropertiesArray(args: CommandArgs): string[] {
-    return args.options.selectProperties
+    const selectProperties = args.options.selectProperties
```

Comment:
Let's use single quotes everywhere for consistency

Reply:
You'd think I wrote this file though. It's everywhere 😅